src/components/sound.js

```diff
@@ -125,24 +131,47 @@ module.exports.Component = registerComponent('sound', {
       evt.detail.cameraEl.getObject3D('camera').add(listener);
     });
 
-    sound = this.sound = new THREE.PositionalAudio(listener);
-    el.setObject3D(this.attrName, sound);
-
-    sound.source.onended = function () {
-      sound.onEnded();
-      el.emit('sound-ended');
-    };
+    this.soundsPoolGroup = new THREE.Group();
+    this.soundsPool = [];
+    // var soundGroup = new THREE.Group();
+    for (var i = 0; i < this.data.pool; i++) {
+      var sound = new THREE.PositionalAudio(listener);
+      this.soundsPool.push(sound);
+      this.soundsPoolGroup.add(sound);
+    }
+    el.setObject3D(this.attrName, this.soundsPoolGroup);
 
-    return sound;
+    this.soundsPool.forEach(function (sound) {
+      sound.source.onended = function () {
+        sound.onEnded();
+        el.emit('sound-ended', {index: i});
+      };
+    });
   },
 
   playSound: function () {
-    if (!this.sound.source.buffer) { return; }
-    this.sound.play();
+    for (var i = 0; i < this.soundsPool.length; i++) {
+      var sound = this.soundsPool[i];
+      if (!sound.isPlaying && sound.source.buffer) {
+        this.soundsPool[i].play();
+        return;
+      }
+    }
+    warn('Not unused sound found on pool, please increate its size using the `pool` attribute');
```

Comment:
Maybe:
`All the sounds are playing. If you need to play more sounds simultaneously consider increasing the size of pool with the `poolSize` attribute.`